config/config.sh

```diff
@@ -4,3 +4,5 @@ stow --target=$HOME urxvt
 stow --target=$HOME xresources
 stow --target=$HOME emacs
 stow --target=$HOME git
+stow --target=$HOME qutebrowser
+sudo stow --target=/etc networkmanager
```

Comment:
given that networkmanager config lives in /etc, it can be setup using nixos options, I didn't do it as it will mean we have to maintain two ways two do this, one for arch and one for nixos. However, I think that in the long term this should probably be managed by nixos.

Reply:
agree, let's figure it out later